tests/baselines/reference/infiniteExpansionThroughInstantiation.errors.txt

```diff
@@ -24,20 +24,22 @@ tests/cases/conformance/types/typeRelationships/recursiveTypes/infiniteExpansion
     var list: List<string>;
     var ownerList: OwnerList<string>;
     list = ownerList; 
-    ~~~~
+           ~~~~~~~~~
 !!! error TS2322: Type 'OwnerList<string>' is not assignable to type 'List<string>'.
 !!! error TS2322:   Types of property 'data' are incompatible.
 !!! error TS2322:     Type 'List<string>' is not assignable to type 'string'.
+!!! related TS6230 tests/cases/conformance/types/typeRelationships/recursiveTypes/infiniteExpansionThroughInstantiation.ts:16:8: Did you mean to access the property 'data' of OwnerList<string> instead?

```

Comment:
This probably shouldn't be issued because it's not talking about the original source/target.

Reply:
Actually it is not wrong, `ownerList.data` is valid and would be compatible with `list`, since `OwnerList<string>` is `List<List<U>>`. It will probably make more sense when I switch to using the entity name rather than the type.

Reply:
@DanielRosenwasser  This now reads `Did you mean to access the 'data' property on 'ownerList' instead?` which is both accurate and clear IMO.